docs/datagen/intro.md

```diff
@@ -0,0 +1,51 @@
+Data Generators
+===============
+
+Data generators are a way to programmatically generate the assets and data of your mods. It allows the definition of the contents of these files in your code and the automatic generation of these files, without worrying about the specification of the files.
+
+The data generator system is loaded by the main class `net.minecraft.data.Main`. Different arguments can be passed to it, to customize which mods' data are gathered, what existing files are considered, etc. The class responsible for data generation is `net.minecraft.data.DataGenerator`.
+
+ForgeGradle provides the `runData` task to run the data generators. The IDE-specific runs generation tasks (`gen***Tasks`) also create run configurations for running the data generator.
```

Comment:
There's only two tasks, so you can probably be specific. And the suffix is `Runs`,  not `Tasks`:
`genIntellijRuns` / `genEclipseRuns`

Reply:
I tried to future-proof, because I had this thought that ForgeGradle may support other IDEs in the future. Yeah, I might just list them out to be crystal clear. The suffix part is my mistake though.

Reply:
Don't forget `genVSCodeRuns` then

Reply:
TIL there's a third 😂 

I'm fine either way then, listing all three or keeping `***`

Reply:
Yep, included the VS Code one.